bip/t3dn_bip/utils.py

```diff
@@ -117,7 +106,7 @@ def load_file(filepath: str, max_size: tuple) -> dict:
         ValueError: If file is not BIP and Pillow is not installed.
     '''
     with open(filepath, 'rb') as bip:
-        if bip.read(4) == BIP2_MAGIC:
+        if bip.read(4) == BIP_FORMATS['BIP2'].magic:
```

Comment:
Just for consistency, we should use `MAGIC_LENGTH` and `startswith` as you have implemented above.

Reply:
Fair point, I'll add that and then seek to the right offset (in case `MAGIC_LENGTH` > 4).